tests/Basic/IFrameTest.php

```diff
@@ -14,7 +14,7 @@ public function testIFrame()
         $el = $webAssert->elementExists('css', '#text');
         $this->assertSame('Main window div text', $el->getText());
 
-        $this->getSession()->switchToIFrame('subframe');
+        $this->getSession()->switchToIFrame(0);
```

Comment:
since Selenium 3.x both Chrome and Firefox accepts only iframe index instead of iframe name

passing webdriver element is not supported https://github.com/instaclick/php-webdriver

Reply:
Should we preserve backwards-compatible behavior by using the iframe name on Selenium2 and index on Selenium3?

Reply:
The frame switching support by name/id on Selenium 3 was fixed in the https://github.com/minkphp/MinkSelenium2Driver/pull/382.